aiida_quantumespresso/parsers/parse_raw/pw.py

```diff
@@ -267,7 +267,7 @@ def detect_important_message(logs, line):
             logs.warning.append(message)
 
 
-def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None):
+def parse_stdout(stdout, input_parameters, parser_options=None, parsed_xml=None, settings={}):
```

Comment:
Note that using mutable objects (as a `dict` is) for function defaults is dangerous. The reason is that the default can actually change if it is mutated. Rather, you should define `settings=None` and then in the function body you call `settings = settings or {}` which will assign the empty dict if it is `None`.

Reply:
I see, that makes sense, good to know - thanks!